common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
   "message.voicechat.onboarding.activation.voice.name": "Voice Activation",
   "message.voicechat.onboarding.ptt.title": "Choose a Push to Talk Keybind",
   "message.voicechat.onboarding.ptt.description": "Please select the key that should be used to activate your microphone.",
-  "message.voicechat.onboarding.ptt.button_description": "Press to reassign",
+  "message.voicechat.onboarding.ptt.button_description": "Press again to reassign",
   "message.voicechat.onboarding.voice.title": "Adjust Microphone Activation Threshold",
-  "message.voicechat.onboarding.voice.description": "Enable microphone testing and make sure the volume is below the slider when you're silent and above it when you are speaking.",
+  "message.voicechat.onboarding.voice.description": "Enable microphone testing and make sure the volume is below the slider when you are silent and above it when you are speaking.",
   "message.voicechat.onboarding.final": "Finish Setup",
   "message.voicechat.onboarding.final.description.success": "You have successfully configured your voice chat!\n\nOpen the voice chat menu with %s.",
-  "message.voicechat.onboarding.final.description.ptt": "Hold %s while talking to use the voice chat.",
-  "message.voicechat.onboarding.final.description.voice": "You will be muted when exiting this screen. Press %s to unmute.",
-  "message.voicechat.onboarding.final.description.configuration": "You can always change all settings in the voice chat menu.\nKeybinds can be changed like any other keybind in the game.",
-  "message.voicechat.onboarding.final.finish_setup": "Finish setup",
+  "message.voicechat.onboarding.final.description.ptt": "Hold %s while speaking to use the voice chat.",
+  "message.voicechat.onboarding.final.description.voice": "You will be muted after exiting this screen. Press %s to unmute.",
+  "message.voicechat.onboarding.final.description.configuration": "You can always change all the settings in the voice chat menu.\nKeybinds, however, can be changed just like any other keybind in the game.",
+  "message.voicechat.onboarding.final.finish_setup": "Finish Setup",
   "resourcepack.voicechat.classic_icons": "Classic Icons",
-  "resourcepack.voicechat.classic_icons.description": "Classic Voice Chat Icons",
+  "resourcepack.voicechat.classic_icons.description": "Classic voice chat icons",
   "resourcepack.voicechat.white_icons": "White Icons",
-  "resourcepack.voicechat.white_icons.description": "White Voice Chat Icons",
+  "resourcepack.voicechat.white_icons.description": "White voice chat icons",
   "resourcepack.voicechat.black_icons": "Black Icons",
-  "resourcepack.voicechat.black_icons.description": "Black Voice Chat Icons",
+  "resourcepack.voicechat.black_icons.description": "Black voice chat icons",
   "cloth_config.voicechat.settings": "Voice Chat Settings",
   "cloth_config.voicechat.category.general": "General",
-  "cloth_config.voicechat.category.other": "Other",
+  "cloth_config.voicechat.category.other": "Primary",
```

Comment:
Primary??

Reply:
The word "Other" is often used in software as a synonym for "Miscellaneous" and it's typically placed at the end of a list, as in this case.

Therefore, my intention was to rename it to something that clearly indicates that clicking this button does not lead to any minor or unimportant options. Instead, it closes the Cloth Config configuration screen entirely and opens the main voice chat menu with all the most important options.

In my opinion, this is not a good UX, and I would rather get rid of this button completely and all the logic behind it if it were up to me.

Do you have any suggestions for a new label for this button? Or perhaps you agree with me that it might be better to remove it altogether? Please let me know.

Reply:
"Miscellaneous" would fit better thats true, but primary is very confusing. This basically shows the in game voice chat settings screen, which has a lot of miscellaneous settings that don't really have a clear category

Reply:
By changing this word to something else, I wanted to emphasize that the options found in the main voice chat menu, despite being a mixture of various unrelated things, are still the most essential options that users need.

The word "primary" was the best I could come up with at that time.

Do you have any suggestions?

Reply:
I think people would be confused by primary. I think its also not important to emphasize this category (i even thought about removing it from there).

Reply:
If you say so, then so be it. I'll restore the original version in a moment.

However, please reconsider removing that button completely from there because, as I mentioned in a previous comment, it's a bad UX, it's unintuitive, and, in my opinion, likely to cause more confusion than benefit.

Reply:
You can also rename it to "Miscellaneous" or we think about it again and find a better name that also conveys what you mean with "Primary"

Reply:
I can try to come up with some other ideas. What do you think of the following suggestions?:
- Main options
- Primary options
- Essential options
- Basic options
- Primary menu
- Basic menu

If none of these work for you and you do not have any ideas either, then I'll go back to “Other” or “Miscellaneous” (although the latter is a bit long).

Reply:
Yeah these also sound good. Or maybe we should be really transparent and just call it "in-game menu", what do you think about that?

Reply:
Your suggestion may not be entirely clear to all users either, but in my opinion, it is still better in this regard than what I suggested in my previous comment, and definitely an improvement over the current "other".